editor/src/components/editor/store/editor-state.ts

```diff
@@ -434,7 +438,7 @@ export function getOpenUIJSFileKey(model: EditorState): string | null {
   const openEditorTab = getOpenEditorTab(model)
   if (
     openFile != null &&
-    isUIJSFile(openFile) &&
+    isTextFile(openFile) &&
```

Comment:
This definitely feels like a candidate for some kind of `isParsedTextFile` or something along those lines (something to signify that it's not just a regular text file, but rather a code file that we can parse)

Reply:
Done in adc65a8

Reply:
I loved getting all these emails saying done 😀